src/tests/add-remove.test.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] I can see you have disabled the `eslint` most of the pages. Please try to fix them as they are included just to make your code standard. To fix it you can simply try running `npx eslint . --fix` if on a windows system, that usually resolves it on my end.